### PR TITLE
Answers Lina Kimmig

### PR DESCRIPTION
**Pull request by Lina-Caro**
I hope I'm doing this pull request correctly, haven't done this before.

### CODE REVIEW

File: Material_Part1_PEP/PEP_right_or_wrong.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Which of the below alignments are correct?
         abs_area = area_A + area_B +
           area_C + area_D
 
-- [ ] b
+- [x] b
```
Comment (iulusoy) [inline Material_Part1_PEP/PEP_right_or_wrong.md:11]: d) and e) are also correct

File: Material_Part1_PEP/PEP_right_or_wrong.md [hunk 2]
```diff
@@ -49,28 +49,28 @@ Variables:
...
-- [ ] b `list_ = my_areas`
+- [x] b `list_ = my_areas`
 - [ ] c `__list__ = my_areas`
 - [ ] d `_list = my_areas`  
 
 Modules:  
 - [ ] a `Numerical.py`
-- [ ] b `numerical.py`
+- [x] b `numerical.py`
 - [ ] c `numerical_analysis.py`
```
Comment (iulusoy) [inline Material_Part1_PEP/PEP_right_or_wrong.md:69]: c) is also correct

File: Material_Part1_PEP/example1.py [hunk 3]
```diff
@@ -19,16 +19,18 @@ def find_files(path=None, pattern="*.png", recursive=True, limit = 20) -> list:
...
+    return result
+
 
-if __name__=="__main__":
-    list = find_files(path="./data/")
-    print("Found files {}".format(list))
\ No newline at end of file
+if __name__ == "__main__":
+    list_ = find_files(path="./data/")
+    print("Found files {}".format(list_))
```
Comment (iulusoy) [inline Material_Part1_PEP/example1.py:36]: looks good!

File: Material_Part2_Linter/example2.py [hunk 4]
```diff
@@ -1,19 +1,21 @@
...
+def area_circ(r_in):
     """Calculate the area of a circle with given radius.
 
     :Input: The radius of the circle (float, >=0).
     :Returns: The area of the circle (float)."""
-    if r_in<0:
+    if r_in < 0:
         raise ValueError("The radius must be >= 0.")
-    Kreis=np.pi*r_in**2
+    circle = np.pi * r_in ** 2
```
Comment (iulusoy) [inline Material_Part2_Linter/example2.py:11]: nice!

**Review by iulusoy — COMMENTED** [review]
Nicely done!

**Comment by iulusoy** [comment]
After you saw my comments, you can just close the PR on your side.

**Comment by Lina-Caro** [comment]
Thank you for the great course! Although I already knew some of it, I learned a lot. I hope you will offer this course again, I will tell my colleagues and friends about it.

**Pull request closed without merging**